pkg/monitortestlibrary/allowedalerts/basic_alert.go

```diff
@@ -111,6 +112,11 @@ func (a *alertBuilder) neverFail() *alertBuilder {
 	return a
 }
 
+func (a *alertBuilder) alwaysFlake() *alertBuilder {
+	a.allowanceCalculator = alwaysFlake()
+	return a
+}
+
```

Comment:
The use of this was removed, but I left the alwaysFlake functionality in so it's visible when we start working on more alerts to help SD out.